src/xdocs/config_misc.xml

```diff
@@ -1260,6 +1261,24 @@ void foo(String aFooString,
         </p>
       </subsection>
 
+      <subsection name="Notes" id="NewlineAtEndOfFile_Notes">
+        <p>
+          This will check against the platform-specific default line separator.
+        </p>
+        <p>
+          It is also possible to enforce the use of a specific line-separator
+          across platforms, with the <code>lineSeparator</code> property.
+        </p>
+        <p>Valid values for the <code>lineSeparator</code> property are:</p>
```

Comment:
Do we really need this when we have `property_types.html#lineSeparator`?

Reply:
Removed.